examples/mobile/UIComponents/components/appbar/actionButton/actionButtonText.html

```diff
@@ -13,20 +13,19 @@
 	<div class="ui-page" id="simpleTitleBack">
 		<header>
 			<div class="ui-appbar-left-icons-container">
-				<button class="ui-btn ui-btn-icon ui-btn-icon-back" data-rel="back"></button>
+				<button class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></button>
 			</div>
 			<div class="ui-appbar-title-container">
-				<span class="ui-appbar-title">Application title</span>
+				<span class="ui-appbar-title">Sample title</span>
 			</div>
 			<div class="ui-appbar-action-buttons-container">
-					<button class="ui-btn ui-btn-icon ui-btn-icon-add"></button>
-					<button class="ui-btn ui-btn-icon ui-btn-icon-search"></button>
-					<button class="ui-btn ui-btn-icon ui-btn-icon-more"></button>
+					<button class="ui-btn" data-style="flat">Menu</button>
```

Comment:
Press effect for `Menu` has wrong sizes:
- height: 48px
- border-radius: 24px

Reply:
Right. Fixed.